conf/mta-sts.txt

```diff
@@ -0,0 +1,4 @@
+version: STSv1
+mode: testing
```

Comment:
Should document somewhere (prob in web.sh) the difference between enforced/testing/none
https://www.uriports.com/blog/mta-sts-explained/

Reply:
I added the url above, but what do you propose?

Reply:
I added comments to the web.sh

# Create a generic mta-sts.txt file which is exposed via the
# nginx configuration at /.well-known/mta-sts.txt
# more documentation is available on: 
# https://www.uriports.com/blog/mta-sts-explained/
# default mode is "testing", which means: "Messages will be delivered as 
# though there was no failure but a report will be sent if TLS-RPT is configured"
# other valid modes are: "enforce" and "none".